tests/features/start.feature

```diff
@@ -1,16 +1,18 @@
 Feature: Start server
     Scenario: I connect to server control
+        Given ServerControl is up
+        And I have no errors
         When I connect to service control
         Then I have no errors
 
     Scenario: Ping works
-        And I ping to the server
-        Then I have no errors
-
-    Scenario: I can install server
-        When I install "1.20.6-50.1.3" server version
+        When I ping to the server
         Then I have no errors
 
     Scenario: I can start server
         When I start server
         Then I have no errors
+
+    Scenario: I can install server
```

Comment:
May be i dont understand something, but should not we install server before we try to start it

Reply:
Oh, yeah, we should

Reply:
wait for fixes :)